jina/helloworld/multimodal/app.py

```diff
@@ -63,10 +63,8 @@ def hello_world(args):
         f.index(inputs=from_csv(fp), request_size=10, show_progress=True)
 
     # search it!
+    os.environ['PORT_EXPOSE'] = str(args.port_expose)
```

Comment:
why this change, if the `Flow` did not enter its context it is still safe to adapt